common/djangoapps/student/tests/test_create_account.py

```diff
@@ -32,6 +32,22 @@
 TEST_CS_URL = 'https://comments.service.test:123/'
 
 
+def path_decorator(patch_string):
```

Comment:
Please document decorator description it does not look like some generic thing, we need to understand in which cases it can be used.